pachyderm/templates/pachd/tls-secret.yaml

```diff
@@ -1,16 +1,16 @@
+{{- if .Values.tls.create }}
 {{- /*
 SPDX-FileCopyrightText: Pachyderm, Inc. <info@pachyderm.com>
 SPDX-License-Identifier: Apache-2.0
 */ -}}
-{{ if and .Values.tls.crt .Values.tls.key }}
 apiVersion: v1
 data:
-  tls.crt: {{ .Values.tls.crt | b64enc | quote }}
-  tls.key: {{ .Values.tls.key | b64enc | quote }}
+  tls.crt: {{ .Values.tls.create.crt | b64enc | quote }}
+  tls.key: {{ .Values.tls.create.key | b64enc | quote }}
 kind: Secret
 metadata:
   labels:
-    app: pachd-tls-cert
```

Comment:
"App" should actually be pachyderm (labels are generally used to isolate installs of a given application on a cluster)